src/inmanta/protocol/methods_v2.py

```diff
@@ -313,7 +324,12 @@ def environment_settings_list(tid: uuid.UUID) -> model.EnvironmentSettingsRepons
 )
 def environment_settings_set(tid: uuid.UUID, id: str, value: model.EnvSettingType) -> ReturnValue[None]:
     """
-    Set a value
+    Set or update a specific setting in an environment's configuration.
```

Comment:
I don't really see what the difference would be between set and update.
```suggestion
    Set a specific setting in an environment's configuration.
```